### PR TITLE
Add an option that allows you to disable drag and drop

### PR DESCRIPTION
**Pull request by andreiglingeanu**
This pull request add `drag_and_drop` option to every builder that allows you to disable drag and drop for this particular builder. This is handy if you need a more complicated drag and drop behavior or if you want to switch you drag and drop library.

It also refactors `builder.js` a little allong the way.

### CODE REVIEW

File: includes/option-types/builder/extends/class-fw-option-type-builder.php [hunk 1]
```diff
@@ -181,6 +181,7 @@ private function fix_base_defaults($option = array())
 			 * scroll up -> add element -> scroll down -> configure it -> scroll up -> ...
 			 */
 			'fixed_header' => false,
+			'drag_and_drop' => true
```
Comment (ghost) [inline includes/option-types/builder/extends/class-fw-option-type-builder.php:184]: Describe what this parameter means

File: includes/option-types/builder/static/js/builder.js [hunk 2]
```diff
@@ -358,6 +358,14 @@ jQuery(document).ready(function($){
 							return this;
 						},
 						initSortable: function(){
+							var hasDragAndDrop = builder.rootItems.view.$el
```
Comment (moldcraft) [inline includes/option-types/builder/static/js/builder.js]: No `;` at the end and too many new lines

**Comment by ghost** [comment]
nvm, I'll do minor fixes

File: includes/option-types/builder/static/js/initialize-builder.js [hunk 3]
```diff
@@ -0,0 +1,488 @@
...
+	};
+
+
+	return {
+		init: init
+	};
+
+	/**
+	* Loop recursive through all items in given collection
+	*/
```
Comment (ghost) [inline includes/option-types/builder/static/js/initialize-builder.js:111]: Missing a space in comment, should be

```
/**
 * Loop recursive through all items in given collection
 */
```

**Comment by andreiglingeanu** [comment]
@moldcraft Thanks for fixing it by yourself. Is there something more I need to do?

**Comment by ghost** [comment]
Submit a PR with description for the `'drag_and_drop'` param, what it does, in which cases it is needed/useful

**Comment by andreiglingeanu** [comment]
@moldcraft #11.

**Pull request merged**